node/flatpak-node-generator.py

```diff
@@ -668,7 +688,7 @@ def process_lockfile(self, lockfile: Path) -> Iterator[Package]:
         raise NotImplementedError()
 
 
-class ModuleProvider(contextlib.AbstractContextManager):
+class ModuleProvider(contextlib.AbstractContextManager['ModuleProvider']):
```

Comment:
Looks like this broke compatibility with Python ≤3.8 (including the one in fd.o sdk 20.08):
```
Traceback (most recent call last):
  File "../tools/node/flatpak-node-generator.py", line 691, in <module>
    class ModuleProvider(contextlib.AbstractContextManager['ModuleProvider']):
TypeError: 'ABCMeta' object is not subscriptable
```